src/sentry/api/endpoints/group_autofix_setup_check.py

```diff
@@ -30,28 +28,31 @@
 from rest_framework.request import Request
 
 
-def get_autofix_integration_setup_problems(organization: Organization) -> str | None:
+def get_autofix_integration_setup_problems(
+    organization: Organization, project: Project
+) -> str | None:
     """
     Runs through the checks to see if we can use the GitHub integration for Autofix.
 
     If there are no issues, returns None.
     If there is an issue, returns the reason.
     """
-    integrations = integration_service.get_organization_integrations(
+    organization_integrations = integration_service.get_organization_integrations(
         organization_id=organization.id, providers=["github"], limit=1
     )
 
-    integration = integrations[0] if integrations else None
+    organization_integration = organization_integrations[0] if organization_integrations else None
+    integration = organization_integration and integration_service.get_integration(
+        organization_integration_id=organization_integration.id
+    )
+    installation = integration and integration.get_installation(organization_id=organization.id)
 
-    if not integration:
+    if not installation:
         return "integration_missing"
 
-    if integration.status != ObjectStatus.ACTIVE:
-        return "integration_inactive"
+    code_mappings = get_sorted_code_mapping_configs(project)
```

Comment:
Oh didn't know this util existed, should be using it for how the other autofix endpoints gets the code mappings too. I'll make a quick refactor